README.md

```diff
@@ -434,6 +434,47 @@ chatSession.onTyping(event => {
 Subscribes an event handler that triggers whenever a `application/vnd.amazonaws.connect.event.typing` event is created by any participant.
 The `data` field has the same schema as `chatSession.onMessage()`.
 
+
+#### `chatSession.onParticipantIdle()`
+```js
+chatSession.onParticipantIdle(event => {
+  const { chatDetails, data } = event;
+  if (data.ParticipantRole === "AGENT") {
+    // ...
+  }
+});
+```
+#### `chatSession.onParticipantReturned()`
+```js
+chatSession.onParticipantReturned(event => {
+  const { chatDetails, data } = event;
+  if (data.ParticipantRole === "AGENT") {
+    // ...
+  }
+});
+```
+#### `chatSession.onAutoDisconnection()`
+```js
+chatSession.onAutoDisconnection(event => {
+  const { chatDetails, data } = event;
+  if (data.ParticipantRole === "AGENT") {
+    // ...
+  }
+});
+```
+`onParticipantIdle`, `onParticipantReturned`, and `onAutoDisconnection` are related to [set up chat timeouts for chat participants](https://docs.aws.amazon.com/connect/latest/adminguide/setup-chat-timeouts.html).
+The `data` field follows the schema of the [`Item` data type](https://docs.aws.amazon.com/connect-participant/latest/APIReference/API_Item.html), with an additional field for `InitialContactId`.  The following fields(for `onParticipantIdle`) are present:
```

Comment:
instead of mentioning this way - can we follow JSDoc syntax ; this cheatsheet is helpful for referencing https://docs.joshuatz.com/cheatsheets/js/jsdoc/

Lets make it a pattern on all new fn we expose to public packages.